Makefile

```diff
@@ -167,16 +167,8 @@ PROJECTS := dome.bin modules
 .PHONY: all clean reset cloc $(PROJECTS)
 
 all: $(PROJECTS)
-
-WREN_LIB ?= $(LIBS)/libwren.a
-WREN_PARAMS ?= $(ARCH) WREN_OPT_RANDOM=0 WREN_OPT_META=1   
-$(LIBS)/wren/lib/libwren.a:
-	@echo "==== Cloning Wren ===="
-	git submodule update --init -- $(LIBS)/wren
-$(LIBS)/wren: $(LIBS)/wren/lib/libwren.a
-$(WREN_LIB): $(LIBS)/wren
-	@echo "==== Building Wren ===="
-	./scripts/setup_wren.sh $(WREN_PARAMS)
+WREN_LIB ?= $(OBJS)/libwren.o
+WREN_PARAMS ?= -DWREN_OPT_RANDOM=0 -DWREN_OPT_META=1
```

Comment:
I'd probably move these to near the top of the file, and then we can append `-DDEBUG=1` to the `WREN_PARAMS` variable if we are compiling in debug mode.